stdlib/3.4/enum.pyi

```diff
@@ -20,8 +21,6 @@ class Enum:
 class IntEnum(int, Enum):
     value = ...  # type: int
 
-_T = TypeVar('_T')
-
 def unique(enumeration: _T) -> _T: ...
```

Comment:
Oh, that's not how it's supposed to be used. As a class decorator it does indeed work as expected. Not sure why or how.